ReadMe.md

```diff
@@ -87,6 +87,7 @@ To copy and paste | コピペ（する） | |
 English | Japanese | Kana | Notes
 --------|----------|------|------
 Device, terminal | 端末 | たんまつ | Smartphones, tablets, e-readers, command-line interfaces, etc
+Cell phone | ケータイ / 携帯電話　| けいたいでんわ | standard (non-smartphone) cell phone　
```

Comment:
Oh I totally got your point with the Smartphones. Sorry about that. Yes I should indeed omit the (non-smartphone) stuff. Sorry about that.